android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerViewManager.java

```diff
@@ -610,11 +615,15 @@ public void receiveCommand(final ReactTVExoplayerView root, int commandId, @Null
                     root.resumeTo(positionMs);
                 }
                 break;
-            case COMMAND_SEEK_TO_POSITION:
+            case COMMAND_SEEK_TO_RESUME_POSITION:
                 long resumeMs = args.getInt(0) * 1000;
                 Log.d(WebUtil.DEBUG, "resumeToPosition " + resumeMs);
                 root.resumeTo(resumeMs);
                 break;
+            case COMMAND_SEEK_TO_POSITION:
+                long seekToMs = args.getInt(0) * 1000;
+                root.seekTo(seekToMs);
+                break;
```

Comment:
Not sure why playback time is measured in seconds on the JS side when it supports seek in milliseconds for the player side. Seeking by rounding to seconds loses a lot of accuracy, and I've noticed that mobile and tv are both using seconds in JS side.

Reply:
Not sure why either, it could be that the backend sends back these position in seconds.